packages/ember-glimmer/lib/utils/references.js

```diff
@@ -75,9 +75,12 @@ class PropertyReference extends CachedReference { // jshint ignore:line
     let { _parentReference, _parentObjectTag, _propertyKey } = this;
 
     let parent = _parentReference.value();
-    _parentObjectTag.update(tagFor(parent));
 
-    return get(parent, _propertyKey);
+    if (parent) {
+      _parentObjectTag.update(tagFor(parent));
+
+      return get(parent, _propertyKey);
+    }
```

Comment:
I confirmed locally that updating again works. I'm not sure if you would need to update the tag because compute would be idempotent operation until you have a truthy value. That brings up another point, can the path ref here ever evaluate to something that is not getable e.g. `true`?